cmd/config.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long:  `Modifies Odo specific configuration settings within the config file.`,
+	Example: fmt.Sprintf("%s\n%s\n",
+		configurationGetCmd.Example,
+		configurationSetCmd.Example),
+	Aliases: []string{"configuration"},
+	// 'odo config' is the same as 'odo config get'
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 && args[0] != "get" && args[0] != "set" {
+			return fmt.Errorf("Unknown command, use set or get")
+		}
+		return nil
+	}, Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 && args[0] == "set" {
+			configurationSetCmd.Run(cmd, args)
+		} else if len(args) > 0 && args[0] == "get" {
+			configurationGetCmd.Run(cmd, args)
+		} else {
+			configurationGetCmd.Run(cmd, args)
+		}
+	},
+}
+
+var configurationSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a value in odo config file",
+	Long: `Set an individual value in the Odo configuration file 
+ # Available Parameters:
+ UpdateNotification - holds the status of update checker(true or false)`,
```

Comment:
this sounds like it is holding whatever update is required or not.
How about: "Controls if an update notification is shown or not" @cdrage might help us with formulating something better ;-)